lib/util/template/index.ts

```diff
@@ -231,6 +231,19 @@ export function compile(
   return handlebars.compile(template)(filteredInput);
 }
 
+export function safeCompile(
+  template: string,
+  input: CompileInput,
+  filterFields = true
+): string {
+  try {
+    return compile(template, input, filterFields);
+  } catch (err) {
+    logger.warn({ err }, 'Error compiling template');
```

Comment:
```suggestion
    logger.warn({ err, template }, 'Error compiling template');
```
so we know which template caused the error?